doc/orchestrator.rst

```diff
@@ -165,3 +165,399 @@ A full example of configuring KeyDB to run in SmartSim is shown below
   export REDIS_CONF=/path/to/keydb.conf
 
   # run smartsim workload
+
+Multiple Orchestrators
+======================
+
+SmartSim offers functionality to automate the deployment of multiple
+databases on an HPC cluster, supporting workloads that require multiple
+``Orchestrators`` for a ``Experiment``. For instance, a workload may consist of a
+simulation with high inference performance demands, along with an analysis and
+visualization workflow connected to the simulation. In such cases, SmartSim
+allows you to augment the number of clustered orchestrators or provision multiple
+co-located orchestrators, ensuring that the workload can scale to meet its resource
+requirements. Below is a general example demonstrating the process of launching
+multiple databases with unique identifiers and distributing data between them.
+
+.. note::
+
+  To run the following example, configure your environment by following
+  :ref:`Site Installation <site_installation>` then build SmartSim
+  by following the :ref:`Build Steps <from-pypi>`.
+
+Workflow Initialization and Multi Orchestrator Setup
+----------------------------------------------------
+
+Defining workflow stages requires the utilization of functions associated
+with the ``Experiment`` object. The Experiment object is intended to be instantiated
+once and utilized throughout the workflow runtime. Begin by creating a ``Experiment``
+object and assign it to the ``exp`` variable. In this example, we instantiate the
+``Experiment`` object with the name ``getting-started-multidb``. We setup the SmartSim
+``logger`` as well for use within the Experiment. For more information on the SmartSim,
+click here.
```

Comment:
I think this sentence -- "For more information on the SmartSim, click here" -- got mangled in editing. Fix or delete